core/src/main/java/hudson/model/Items.java

```diff
@@ -236,7 +235,7 @@ public static String toNameList(Collection<? extends Item> items) {
         StringTokenizer tokens = new StringTokenizer(list, ",");
         while (tokens.hasMoreTokens()) {
             String fullName = tokens.nextToken().trim();
-            if (StringUtils.isNotEmpty(fullName)) {
+            if (fullName != null && !fullName.isEmpty()) {
```

Comment:
could we create our own method for this, it was easier to read before this?

Reply:
`Util#fixEmpty` would allow
```suggestion
            if (Util.fixEmpty(fullName) != null) {
```

Reply:
agree with @timja definitely easier to read.